### DISCUSSION TITLE
Proof, student ID

### DISCUSSION THREAD

**Discussion by pbarni** [community/community#113341] [GitHub Education] [answered]
### Hi everyone,

on my student id the expiration date is on the back side so I put the front and the back together, but it gets rejected
what to do?

**Comment by SebastienXia** [comment]
## Hi, @pbarni

You've other options for verification such as your 2023-2024 school registration and others.

**Comment by samus-aran** [comment, marked as answer]
Hi @pbarni 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.


- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
